checkers-app/src/components/vote/CommunityNoteCategories.tsx

```diff
@@ -73,6 +100,19 @@ export default function CommunityNoteCategories(Prop: PropType) {
           </Button>
         </>
       ))}
+
+      <div className="relative w-full min-w-[200px]">
+          <textarea
+            rows={6}
+            value={commentOnNote}
+            onChange={handleInputChange}
```

Comment:
See if this works instead? Then can get rid of extra state.

```suggestion
            value={Prop.commentonNote}
            onChange={Prop.onCommentOnNote}
```